src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -1,24 +1,24 @@
 //ie11 fixes
-:host-context([theme="dark"]) {
+:host([theme="dark"]) {
   @include calcite-theme-dark();
 }
 
 // scale variants for child
-:host-context([scale="s"]) {
+:host([scale="s"]) {
```

Comment:
@paulcpederson what about these?

Reply:
You can either leave them, or remove them all together, but changing them to `:host` is not going to do anything. Normal browsers will pick up on the parent component's `:host` styles, so this variable will already be defined on the child. You can see where they are defined on the parent for accordion [here](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-accordion/calcite-accordion.scss#L6-L25) (notice how it's an exact duplication but in `host-context` in the child.

Leaving them as `:host-context` may be better for now so that the scales still work in ie11. They are a much smaller set, so the downside is way more limited.